pyof/v0x01/common/flow_match.py

```diff
@@ -119,3 +120,27 @@ def __init__(self, wildcards=None, in_port=None, dl_src=None, dl_dst=None,
         self.nw_dst = nw_dst
         self.tp_src = tp_src
         self.tp_dst = tp_dst
+
+        if self.wildcards is None:
```

Comment:
@beraldoleal , since we are building a *wildcards automatic calculator method*, should we remove it (wildcard attribute) from the ``__init__()``? And, if so, should we call the ``fill_wildcards`` method on the beginning of the ``pack`` method to update it?